### PR TITLE
Revert "Remove the stickers target (temporarily!)" + fix issues

### PR DESCRIPTION
**Pull request by osdnk**
Reverts rainbow-me/rainbow#1548

### CODE REVIEW

File: ios/Rainbow Stickers/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>$(MARKETING_VERSION)</string>
```
Comment (skylarbarrera) [inline ios/Rainbow Stickers/Info.plist:20]: Where does this pull from? Want to make sure I update correctly on release

**Review by skylarbarrera — APPROVED** [review]
👌

File: ios/Rainbow Stickers/Info.plist (hunk 1 above)
Comment (osdnk) [reply ios/Rainbow Stickers/Info.plist:20]: you just need to edit this from XCode. It has been moved to some variable, but it doesn't matter

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**